models/users.js

```diff
@@ -829,6 +829,26 @@ const fetchUsersListForMultipleValues = async (documentKey, valueList) => {
   }
 };
 
+const updateUsernameToLowercase = async () => {
```

Comment:
the updateUserNameToLowercasePromises array may contain rejected promises if any of the addOrUpdate() functions fail. If this happens, the Promise.all() function will reject, and the return updateUserNameToLowercasePromises.length; statement will never be executed.